src/settings-ui/Settings.UI/Views/ColorPickerPage.xaml

```diff
@@ -127,11 +127,7 @@
                                         <Button.Flyout>
                                             <MenuFlyout>
                                                 <MenuFlyoutItem IsEnabled="{x:Bind CanMoveUp}" Icon="Up" x:Uid="MoveUp" Click="ReorderButtonUp_Click"/>
-                                                <MenuFlyoutItem IsEnabled="{x:Bind CanMoveDown}" x:Uid="MoveDown" Click="ReorderButtonDown_Click">
-                                                    <MenuFlyoutItem.Icon>
-                                                        <FontIcon Glyph="&#xF0AE;" />
-                                                    </MenuFlyoutItem.Icon>
-                                                </MenuFlyoutItem>
+                                                <MenuFlyoutItem IsEnabled="{x:Bind CanMoveDown}" Icon="Down" x:Uid="MoveDown" Click="ReorderButtonDown_Click"/>
```

Comment:
That's strange. Does it say why? I just used the same style as "Up" already was...

Reply:
Also #19080

Reply:
It's because the "Down" icon is not recognized. It's one of the reasons we use glyph codes everywhere, the Symbol names (as listed on the Docs) do not always seem to work.

Reply:
Fine, that'll let me fix _this_ inconsistency while I'm at it...
![image](https://user-images.githubusercontent.com/65828559/185793899-669ddcc9-c101-42db-a4a1-f74db73da7da.png)
 Can I use `<MenuFlyoutItem Icon=E74B"...` here or will that fail too?

Reply:
Use <FontIcon ../> just to be sure.